app/pb/api.proto

```diff
@@ -139,6 +139,11 @@ enum SmokingLocation {
   SMOKING_LOCATION_NO = 5;
 }
 
+message NullableInt32Value {
+  bool is_null = 1;
+  uint32 value = 2;
+}
+
```

Comment:
These could be moved to a helper file if they start to be reused across files.

Also, this is now `uint32` even though it says `int32`!

Reply:
solved